pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -411,6 +416,25 @@ func (r *MemberRollReconciler) reconcileObject(ctx context.Context, roll *maistr
 	return reconcile.Result{}, nil
```

Comment:
Since you are changing this function, what do you think about refactoring this line?
```suggestion
	return common.Reconciled()
```